### ISSUE TITLE
Draft an org level readme?

### ISSUE THREAD

**Issue by mnyrop** [minicomp/maintainer-playbook#7]
See: https://github.com/nyu-dh

**Comment by elotroalex** [comment]
Hm. For minicomp?

**Comment by mnyrop** [comment]
Yep, just so we can add a tiny preface/tagline/status report and some links (e.g., the wiki, the survey) to treat https://github.com/minicomp/ like a concierge "Homepage" of sorts...?

Edit: feel free to reassign back to me if you aren't opinionated about it!

**Comment by elotroalex** [comment]
Ok, cool. I'll put it on the cue.
